### PR TITLE
Implement TTL for DynamoDBChatMessageHistory

### PR DESCRIPTION
**Pull request by brnaba-aws**
- **Description:** Implement TTL for DynamoDBChatMessageHistory, 
  - **Issue:** see #15477,
  - **Dependencies:** N/A,

### CODE REVIEW

**Comment by brnaba-aws** [comment]
@hwchase17  Can you please help me merge this PR?

**Review by baskaryan — COMMENTED** [review]
cc @3coins

**Comment by brnaba-aws** [comment]
@hwchase17  @baskaryan  @3coins 
Any chance to review this?

File: libs/community/langchain_community/chat_message_histories/dynamodb.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ class DynamoDBChatMessageHistory(BaseChatMessageHistory):
             This may also contain global and local secondary index keys.
         kms_key_id: an optional AWS KMS Key ID, AWS KMS Key ARN, or AWS KMS Alias for
             client-side encryption
+        ttl: Optional Time-to-live (TTL) allows you to define a per-item expiration
+            timestamp that indicates when an item is no longer needed.
+            DynamoDB automatically deletes expired items within a few days of their 
+            expiration time, without consuming write throughput.
```
Comment (3coins) [inline libs/community/langchain_community/chat_message_histories/dynamodb.py:44]:
```suggestion
        ttl: Optional Time-to-live (TTL) in seconds. Allows you to define a per-item 
            expiration timestamp that indicates when an item can be deleted from the 
            table. DynamoDB handles deletion of expired items without consuming 
            write throughput. To enable this feature on the table, follow the
            [AWS DynamoDB documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html)
```

**Review by 3coins — COMMENTED** [review]
@brnaba-aws
Thanks for working on this. Minor suggestion on the docs, code looks good otherwise. Can you also fix the lint errors on the CI.

**Comment by brnaba-aws** [comment]
@3coins, Thanks. I fixed the lint issue.

**Comment by baskaryan** [comment]
thank you @brnaba-aws @3coins!

**Pull request merged**